community.dbuild

```diff
@@ -984,4 +989,60 @@ build += {
     extra.exclude: ["plugin"]  // we never build sbt plugins
   }
 
+  // forked to remove the usual bintray gunk
```

Comment:
Can we provide some additional configuration on our side (e.g. `-Dsbt.prohibit.publish=true`) to make sure you don't need to maintain your fork?